### PR TITLE
feat: return component

### PR DESCRIPTION
**Pull request by rbadici-plenty**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/api-client/src/api/getOrder/index.ts [hunk 1]
```diff
@@ -31,3 +31,19 @@ export async function executePayment(context: Context, orderId: number, paymentI
 
   return data;
 }
+
+export async function makeOrderReturn(context: Context, orderId: number, orderAccessKey:string, variationIds: object, returnNote: string): Promise<CreateReturnResponse> {
```
Comment (maxiroellplenty) [inline packages/api-client/src/api/getOrder/index.ts:35]: All the params should only be one object named params of type MakeOrderReturnParams that you have to create. This makes the code more readable.

**Comment by maxiroellplenty** [comment]
run: yarn lint --fix to fix linter issues

**Review by maxiroellplenty — CHANGES_REQUESTED** [review]

File: packages/api-client/src/api/getOrder/index.ts (hunk 1 above)
Comment (maxiroellplenty) [reply packages/api-client/src/api/getOrder/index.ts]: did the changes

**Review by maxiroellplenty — DISMISSED** [review]

File: packages/composables/src/useMakeReturn/index.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+  const makeReturn = async (params: MakeReturnParams): Promise<void> => {
+
+    const returnParams = {
+      orderId: params.orderId,
+      orderAccessKey: params.orderAccessKey,
+      returnNote: params.returnNote
+    };
+
+    // Todo: Json to FormData mapping will be removed one the endpoint supports json payload
```
Comment (ksted) [inline packages/composables/src/useMakeReturn/index.ts:26]:
```suggestion
    // @TODO: Json to FormData mapping will be removed once the endpoint supports json payload
```

**Review by ksted — CHANGES_REQUESTED** [review]
**todo**: Fix lint warnings.

**Review by ksted — APPROVED** [review]

**Pull request merged**